vck-openid/src/commonMain/kotlin/at/asitplus/wallet/lib/oidc/OidcSiopWallet.kt

```diff
@@ -70,9 +70,9 @@ class OidcSiopWallet(
 ) {
     companion object {
         fun newDefaultInstance(
```

Comment:
This PR should be rebased on `develop`, to get rid of these factory methods.

Reply:
done